test/runcheck.sh

```diff
@@ -16,7 +16,7 @@ mkdir -p "$parent/actual"
 if test $kind == p; then
     balFile=`echo "$b" | sed -e 's/.exe/.bal/'`
     $("./$1" >/dev/null 2>"$out")
-    head -n 2 $out | sed -e 's/.*bal:/'"$balFile"':/' | diff -u "$2" -
+    ../../checkPanic.sh "$out" "$balFile" "$2"
```

Comment:
I don't think this is a good approach, because it means checkpanic.sh now has to be run in a particular directory.

Better to have two separate runcheck.sh for now, but put a comment in each saying what the difference is.

Reply:
fixed